package.json

```diff
@@ -28,7 +31,16 @@
     "bs-platform": "^1.7.1"
   },
   "devDependencies": {
-    "reunit": "./reUnit"
+    "reunit": "./reUnit",
+    "opam_of_packagejson": "*"
+  },
+  "opam": {
+    "dependencies": {
```

Comment:
I'm going to merge, but a nice improvement would be to support build dependencies as "devDependencies".